src/sentry/models/groupopenperiod.py

```diff
@@ -25,6 +26,15 @@ class TsTzRange(models.Func):
     output_field = DateTimeRangeField()
 
 
+def should_create_open_periods(type_id: int) -> bool:
+    grouptypes_without_open_periods = options.get(
+        "workflow_engine.group.type_id.should_not_create_open_periods"
+    )
+    if type_id in grouptypes_without_open_periods:
+        return False
+    return True
```

Comment:
should we also introduce a feature flag for this so we can on dial that allows us to disable issue types, and another dial to determine if we want to enable creation of open periods? (my initial thought was we could enable each issue type rather than all at once, but i agree with you that a denylist is a better long term strategy)

my other thought is we can just start with the denylist having all the issue types, then we remove them until only error is left. the only downside to that is we can't do a percentage based rollout on the first issue type.